packages/backend/src/tokens/authObjects.ts

```diff
@@ -200,27 +201,28 @@ const createHasAuthorization =
     orgId,
     orgRole,
     userId,
+    orgPermissions,
   }: {
     userId: string;
     orgId: string | undefined;
     orgRole: string | undefined;
-  }): experimental__CheckAuthorizationWithoutPermission =>
+    orgPermissions: string[] | undefined;
+  }): CheckAuthorizationWithCustomPermissions =>
   params => {
-    if (!orgId || !userId) {
+    if (!params?.permission && !params?.role) {
+      throw new Error('Missing params. `has` from `useAuth` requires a permission or role key to be passed.');
```

Comment:
We're defining a new error message factory in the react package, can we move to `shared` and re-use here and there? Also this message is not accurate in a backend context, there would be no `useAuth`

`useAuthHasRequiresRoleOrPermission`